Makefile

```diff
@@ -7,8 +7,8 @@ SPLUNK_ANSIBLE_BRANCH ?= develop
 SPLUNK_COMPOSE ?= cluster_absolute_unit.yaml
 # Set Splunk version/build parameters here to define downstream URLs and file names
 SPLUNK_PRODUCT := splunk
-SPLUNK_VERSION := 9.1.0.2
-SPLUNK_BUILD := b6436b649711
+SPLUNK_VERSION := 9.0.5.1
+SPLUNK_BUILD := 52d49260b188
```

Comment:
we should not change the version and SHAs as 9.0.5.1 is not the latest Splunk release.